eng/pipelines/runtime.yml

```diff
@@ -118,6 +118,46 @@ jobs:
       - eng/pipelines/mono/*
       - eng/pipelines/libraries/*
 
+#
+# Build CoreLib checked
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/coreclr/templates/build-corelib-job.yml
+    buildConfig: checked
```

Comment:
We don't need the checked corelib job as libraries only build against a release corelib. It runs tests against a checked runtime, but we get corelib from the product build. We should be safe to remove this matrix.

Reply:
fixed.